packages/server/lib/project_utils.ts

```diff
@@ -38,10 +38,8 @@ export const getSpecUrl = ({
 
 export const checkSupportFile = async ({
   supportFile,
```

Comment:
Prob can just be a positional arg, not much point having an object arg for one value.

Reply:
@lmiller1990 You arn't wrong...I do have a sneaky suspicion this will be needed to update this logic to 
```
const found = await fs.pathExists(path.join(configFile.projectPath, supportFile))
```
to handle custom supportFile paths. I'm pretty sure I logged a Jira for this. If not, I'll get it logged.

Reply:
Nope! I'm wrong. I was hitting this issue at one point, but in the latest binary this isn't an issue. 

+1 to updating to a single arg.

Reply:
Done in [84bfdcc](https://github.com/cypress-io/cypress/pull/20643/commits/84bfdcc97c31c9acd6077486f9976bb6b18ddf73)